jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/pojos/geojson/map/MapConnection.java

```diff
@@ -1,80 +1,32 @@
 package us.dot.its.jpo.geojsonconverter.pojos.geojson.map;
 
-import java.util.Objects;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import us.dot.its.jpo.geojsonconverter.DateJsonMapper;
 import us.dot.its.jpo.ode.plugin.j2735.J2735IntersectionReferenceID;
 
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Slf4j
 public class MapConnection {
     private static Logger logger = LoggerFactory.getLogger(MapConnection.class);
 
     private Integer lane;
-	private J2735IntersectionReferenceID remoteIntersection;
-	private Integer signalGroup;
-	private Integer userClass;
-	private Integer connectionID;
-
-    public void setLane(Integer lane) {
-        this.lane = lane;
-    }
-
-    public Integer getLane() {
-        return this.lane;
-    }
-
-    public void setRemoteIntersection(J2735IntersectionReferenceID remoteIntersection) {
-        this.remoteIntersection = remoteIntersection;
-    }
-
-    public J2735IntersectionReferenceID getRemoteIntersection() {
-        return this.remoteIntersection;
-    }
-
-    public void setSignalGroup(Integer signalGroup) {
-        this.signalGroup = signalGroup;
-    }
-
-    public Integer getSignalGroup() {
-        return this.signalGroup;
-    }
-
-    public void setUserClass(Integer userClass) {
-        this.userClass = userClass;
-    }
-
-    public Integer getUserClass() {
-        return this.userClass;
-    }
-
-    public void setConnectionID(Integer connectionID) {
-        this.connectionID = connectionID;
-    }
-
-    public Integer getConnectionID() {
-        return this.connectionID;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (o == this)
-            return true;
-        if (!(o instanceof MapConnection)) {
-            return false;
-        }
-        MapConnection mapConnection = (MapConnection) o;
-        return Objects.equals(lane, mapConnection.lane) && Objects.equals(remoteIntersection, mapConnection.remoteIntersection) && Objects.equals(signalGroup, mapConnection.signalGroup) && Objects.equals(userClass, mapConnection.userClass) && Objects.equals(connectionID, mapConnection.connectionID);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(lane, remoteIntersection, signalGroup, userClass, connectionID);
-    }
+    private J2735IntersectionReferenceID remoteIntersection;
```

Comment:
issue (blocking): I think we should remove this dependency to avoid coupling with the old ODE classes.
suggestion: Use a class from the new POJOs or create an equivalent class in this project.

Reply:
Deleting this whole class. It is not used anymore.